vmServer/accept/execute/action/code/action.ps1

```diff
@@ -0,0 +1,4 @@
+$scriptpath = $MyInvocation.MyCommand.Path
```

Comment:
The code for actions should not live within the server. Can you create another directory outside the server directory tree and store this there? (ideally with paths matching the paths in the sample query protos)

Reply:
Change has been made. The execution happens outside of the server directory now.